### DISCUSSION TITLE
RTSP -> WebRTC/HLS remuxing - The stream doesn't contain any supported codec - V1.4.2 and newer.

### DISCUSSION THREAD

**Discussion by anhorbc** [bluenviron/mediamtx#3804] [Questions]
### Question

Hello! 
I'm trying to upgrade an old mediamtx installation from version 1.1.1 to 1.9.1. When trying to access a WebRTC stream, I get the error "[WebRTC] [session 562309ee] closed: the stream doesn't contain any supported codec, which are currently AV1, VP9, VP8, H264, Opus, G722, G711, LPCM" 

I have tried upgrade version by version and the issue seem to have started in version 1.4.2.  Any ideas?

The source is a rtsp camera. Codec: H264 - MPEG-4 AVC (part 10) (h264), no audio, single stream:
Stream #0:0: Video: h264 (Main), yuvj420p(pc, bt709, progressive), 768x432 [SAR 1:1 DAR 16:9], 30 fps, 30 tbr, 90k tbn
Browser: Microsoft Edge/Chrome

**Comment by aler9** [comment]
This is impossible to debug if you do not provide server logs. If you still have troubles, open a bug report and provide server logs with `logLevel: debug` that clearly show the list of published tracks (and their codecs).
